src/beamlime/dashboard/widgets/job_control_widget.py

```diff
@@ -0,0 +1,239 @@
+
+    def _setup_callbacks(self) -> None:
+        """Set up event handlers for widgets."""
+        self._workflow_select.param.watch(self._on_workflow_changed, 'value')
+
+        # Job action callbacks
+        self._pause_btn.on_click(lambda event: self._on_job_action(JobAction.pause))
+        self._resume_btn.on_click(lambda event: self._on_job_action(JobAction.resume))
+        self._reset_btn.on_click(lambda event: self._on_job_action(JobAction.reset))
+        self._stop_btn.on_click(lambda event: self._on_job_action(JobAction.stop))
+
+        # Global action callbacks
+        self._global_pause_btn.on_click(
+            lambda event: self._on_global_action(JobAction.pause)
+        )
+        self._global_resume_btn.on_click(
+            lambda event: self._on_global_action(JobAction.resume)
+        )
+        self._global_reset_btn.on_click(
+            lambda event: self._on_global_action(JobAction.reset)
+        )
+        self._global_stop_btn.on_click(
+            lambda event: self._on_global_action(JobAction.stop)
+        )
+
+        # Workflow action callbacks
+        self._workflow_pause_btn.on_click(
+            lambda event: self._on_workflow_action(JobAction.pause)
+        )
+        self._workflow_resume_btn.on_click(
+            lambda event: self._on_workflow_action(JobAction.resume)
+        )
+        self._workflow_reset_btn.on_click(
+            lambda event: self._on_workflow_action(JobAction.reset)
+        )
+        self._workflow_stop_btn.on_click(
+            lambda event: self._on_workflow_action(JobAction.stop)
+        )
+
+    def _create_layout(self) -> pn.Column:
+        """Create the main widget layout."""
+        return pn.Column(
+            pn.pane.HTML("<h2>Job Control</h2>"),
+            # Global actions section
+            pn.Column(
+                pn.pane.HTML("Global:"),
+                pn.Row(
+                    self._global_pause_btn,
+                    self._global_resume_btn,
+                    self._global_stop_btn,
+                    self._global_reset_btn,
+                ),
+            ),
+            # Workflow selection section
+            pn.Column(
+                pn.pane.HTML("Workflows:"),
+                self._workflow_select,
+                pn.Row(
+                    self._workflow_pause_btn,
+                    self._workflow_resume_btn,
+                    self._workflow_stop_btn,
+                    self._workflow_reset_btn,
+                ),
+            ),
+            # Job selection and actions section
+            pn.Column(
+                pn.pane.HTML("Jobs:"),
+                self._job_select,
+                pn.Row(
+                    self._pause_btn,
+                    self._resume_btn,
+                    self._stop_btn,
+                    self._reset_btn,
+                ),
+            ),
+            # Status section
+            pn.Column(pn.pane.HTML("Status:"), self._status_text),
```

Comment:
[Minor] I would like to have the status section at the top of the job controller but I guess we can fix that later when we have other bottons working too...

Reply:
This file is removed in a follow-up, see PR description my comments before your review.